package.json

```diff
@@ -5,11 +5,15 @@
   "main": "src/index.ts",
   "types": "src/index.d.ts",
   "scripts": {
-    "startdv": "npm run build && esbuild --bundle examples/DocumentViewer/index.js --loader:.js=jsx --outfile=www/main.js --servedir=www",
-    "startctx": "npm run build && esbuild --bundle examples/Context/index.js --loader:.js=jsx --outfile=www/main.js --servedir=www",
-    "ts-types": " tsc -d --emitDeclarationOnly --outDir lib/context",
     "build": "node ./esbuild.js && npm run ts-types",
-    "preparevwlib": "node copy-webviewer-file.js"
+    "deploydemo": "npm run beforestart && esbuild --bundle examples/Context/index.js --loader:.js=jsx --outfile=www/main.js && firebase deploy --only hosting:pdftron-ychen02",
```

Comment:
I don't think this script should exist in a public repo

Reply:
removed